pkg/geth/node/node_test.go

```diff
@@ -0,0 +1,86 @@
+package node_test
+
+import (
+	"encoding/json"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vulcanize/vulcanizedb/pkg/core"
+	"github.com/vulcanize/vulcanizedb/pkg/geth/node"
+)
+
+type MockNodeFactory struct {
+	gethNodeName   string
+	gethNodeId     string
+	parityNodeName string
+	parityNodeId   string
+	networkId      float64
+	genesisBlock   string
+}
+
+func (mnf MockNodeFactory) NetworkId() float64 {
+	return mnf.networkId
+}
+
+func (mnf MockNodeFactory) GenesisBlock() string {
+	return mnf.genesisBlock
+}
+
+func (mnf MockNodeFactory) GethNodeInfo() (string, string) {
+	return mnf.gethNodeId, mnf.gethNodeName
+}
+
+func (mnf MockNodeFactory) ParityNodeInfo() (string, string) {
+	return mnf.parityNodeId, mnf.parityNodeName
+}
+
+var _ = Describe("Parity Node Info", func() {
+
+	It("Decodes json from parity_versionInfo", func() {
+		var parityNodeInfo core.ParityNodeInfo
+		nodeInfoJSON := []byte(
+			`{
+        "hash": "0x2ae8b4ca278dd7b896090366615fef81cbbbc0e0",
+        "track": "null",
+        "version": {
+          "major": 1,
+          "minor": 6,
+          "patch": 0
+             }
+        }`)
+		json.Unmarshal(nodeInfoJSON, &parityNodeInfo)
```

Comment:
Is this testing `json.Unmarshal`? Perhaps unnecessary, or better tested through whatever function we're exposing